var/spack/repos/builtin/packages/py-matplotlib/package.py

```diff
@@ -11,7 +11,7 @@ class PyMatplotlib(PythonPackage):
     and interactive visualizations in Python."""
 
     homepage = "https://matplotlib.org/"
-    url      = "https://pypi.io/packages/source/m/matplotlib/matplotlib-3.3.2.tar.gz"
+    pypi     = "matplotlib/matplotlib-3.3.3.tar.gz"
```

Comment:
Is this naming scheme constant for pypi? That is, for every package on pypi is the link `https://pypi.io/packages/source/[first-letter]/[name]/[name]-[version].tar.gz`? If so, we could reduce the `pypi` attribute to just be its name on pypi.

```
homepage = "https://matplotlib.org"
pypi = "matplotlib"
```

Reply:
Talked with @adamjstewart on slack, apparently this is not feasible.